src/main/java/io/vertx/ext/mongo/CollationOptions.java

```diff
@@ -15,39 +15,30 @@
  */
 @DataObject(generateConverter = true)
 public class CollationOptions {
-  public static final Boolean DEFAULT_CASE_LEVEL = false;
-  public static final CaseFirst DEFAULT_CASE_FIRST = CaseFirst.off;
-  public static final Alternate DEFAULT_ALTERNATE = Alternate.NON_IGNORABLE;
-  public static final Integer DEFAULT_STRENGTH = 3;
-  public static final Boolean DEFAULT_NUMERIC_ORDERING = false;
-  public static final Boolean DEFAULT_BACKWARDS = false;
-  public static final Boolean DEFAULT_NORMALIZATION = false;
-  public static final MaxVariable DEFAULT_MAX_VARIABLE = MaxVariable.punct;
-  private static final String DEFAULT_LOCALE = Locale.getDefault().toString();
 
   private String locale;
-  private boolean caseLevel;
+  private Boolean caseLevel;
   private CaseFirst caseFirst;
-  private int strength;
-  private boolean numericOrdering;
+  private Integer strength;
+  private Boolean numericOrdering;
   private String alternate;
   private MaxVariable maxVariable;
-  private boolean backwards;
-  private boolean normalization;
+  private Boolean backwards;
+  private Boolean normalization;
 
   /**
    * Default constructor for setting
    */
   public CollationOptions() {
-    this.locale = DEFAULT_LOCALE;
-    caseLevel = DEFAULT_CASE_LEVEL;
-    caseFirst = DEFAULT_CASE_FIRST;
-    strength = DEFAULT_STRENGTH;
-    numericOrdering = DEFAULT_NUMERIC_ORDERING;
-    backwards = DEFAULT_BACKWARDS;
-    maxVariable = DEFAULT_MAX_VARIABLE;
-    normalization = DEFAULT_NORMALIZATION;
-    alternate = DEFAULT_ALTERNATE.toString();
+    locale = null;
+    caseLevel = null;
+    caseFirst = null;
+    strength = null;
+    numericOrdering = null;
+    backwards = null;
+    maxVariable = null;
+    normalization = null;
+    alternate = null;
```

Comment:
These are breaking changes

Reply:
Shall I document it in the wiki and/or move it to another PR (like above's issue with collation vs collationOptions)?